module/VuFindTheme/src/VuFindTheme/Initializer.php

```diff
@@ -440,4 +443,35 @@ protected function updateTranslator($themes)
             }
         }
     }
+
+    /**
+     * Load configured javascript files from mixin.config.php if existing
+     *
+     * @return void
+     */
+    public function loadConfiguredJavascriptFilesFromMixin(): void
+    {
+        $config = $this->serviceManager->get('config');
+        $templatePathStack = $config['view_manager']['template_path_stack'] ?? false;
+        if ($templatePathStack) {
+            /* @var HeadScript $headScript */
+            $headScript = $this->serviceManager->get('ViewHelperManager')
```

Comment:
What is the circumstance under which we would be pulling the HeadScript helper from the primary service manager rather than the view helper manager? Is that case really needed? Also, if you want to be more explicit, you could use `HeadScript::class` instead of `'headScript'` as the service name you fetch from the container, which might be useful for static analysis purposes.